src/Nethermind/Nethermind.Consensus/Scheduler/BackgroundTaskScheduler.cs

```diff
@@ -56,7 +56,7 @@ public BackgroundTaskScheduler(IBlockProcessor blockProcessor, int concurrency,
         _blockProcessor.BlocksProcessing += BlockProcessorOnBlocksProcessing;
         _blockProcessor.BlockProcessed += BlockProcessorOnBlockProcessed;
 
-        _tasksExecutors = Enumerable.Range(0, concurrency).Select(_ => Task.Factory.StartNew(StartChannel)).ToArray();
+        _tasksExecutors = Enumerable.Range(0, concurrency).Select(_ => Task.Factory.StartNew(StartChannel, TaskCreationOptions.LongRunning)).ToArray();
```

Comment:
I remember discussion where we decided not to use `LongRunning`

Reply:
Yea, I'm thinking that this may be the cause of random failure in test. could be wrong though.

Reply:
Is pointless, creates and adds a thread that immediately gets released on first `await`; so LongRunning just adds some additional overhead (thread creation) for async methods

Reply:
Removed